help/export-your-organization.md

```diff
@@ -89,6 +98,37 @@ agreements and corporate policy for this type of export are in place. Note
 that many countries have laws that require employers to notify employees of
 their use of such an export.
 
+The output of this export is suitable to import into an on-premise
+installation of Zulip. It is not designed to be human-readable.
+
+## Compliance export
+
+{!owner-only.md!}
+
+This export is limited to paid Zulip Cloud Standard customers, though
+in rare cases may be available to other organizations in case of due
+legal process. As with full exports without member consent, you will
+also need to email us evidence that you have authority to read
+members' direct messages.
+
+To start this export, email support@zulip.com with your zulip.com URL,
+asking for a compliance export.
+
+You should also tell us the what limits you would like on the export,
+and what output format you would prefer (CSV or JSON). Currently, we
+can provide filters for:
```

Comment:
see note about voice above

Reply:
Also, to describe what goes in the email, how about using an instruction block similar to what we have in https://zulip.com/help/import-from-slack#import-your-data-into-zulip?